src/highlight/HighlightList.scss

```diff
@@ -6,7 +6,7 @@
     height: 100%;
 
     .ba-HighlightSvg.is-listening {
-        .ba-HighlightTarget-rect {
+        .ba-HighlightTarget {
```

Comment:
Is this addressing a bug? If so, it should be a separate commit.

Reply:
It's not a bug for actual users, because they can click `.ba-HighlightTarget-rect` then it bubbles up to `.ba-HighlightTarget`. However, for cypress tests, we use the selector `.ba-HighlightTarget` instead of `.ba-HighlightTarget-rect`, so we cannot click.

Reply:
Hmm... that will break the CSS in preview sdk, can we not just change the selector in cypress tests?

Reply:
I see. Will do